README.md

```diff
@@ -97,11 +95,7 @@ By default metrics are gathered once per second.
 
 Example of running Snap ceph perf counters collector and writing data to file.
 
-Run the Snap daemon on each node with defaults settings:
-```
-$ $SNAP_PATH/snapteld -l 1 -t 0
-```
-Or set custom settings in Snap Global Config in Ceph section (see examples/configs/snap-config-sample.json):
```

Comment:
What do you think about leaving the mention about the possibility of running this plugin with defaults? Maybe it should be rephrased, but in my opinion it should be clear that if your ceph cluster has default settings (I mean ceph_path, path to sockets, etc.), you do not need run snapteld with customize  global config.